src/provider/model/Model.ts

```diff
@@ -175,13 +175,11 @@ export class Model {
         }, []);
     }
 
-    private onWindowSeen(identity: Identity): void {
-        this.getOrCreateExpectedWindow(identity);
```

Comment:
I think we want to keep this call as the first thing the remaining seen/created listener does